examples/jaeger/src/main/java/io/opentelemetry/example/JaegerExample.java

```diff
@@ -0,0 +1,76 @@
+package io.opentelemetry.example;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.opentelemetry.exporters.jaeger.JaegerGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.TracerSdkFactory;
+import io.opentelemetry.sdk.trace.export.SimpleSpansProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
+
+public class JaegerExample {
+  // Jaeger Endpoint URL and PORT
+  String ip; // = "jaeger";
+  int port; // = 14250;
+  // OTel API
+  Tracer tracer;
+  // Create a channel towards Jaeger end point
+  ManagedChannel jaegerChannel;
+  // Export traces to Jaeger
+  JaegerGrpcSpanExporter jaegerExporter;
+
+  private void initTracer() {
+    // Create a channel towards Jaeger end point
+    jaegerChannel = ManagedChannelBuilder.forAddress(ip, port).build();
+    // Export traces to Jaeger
+    jaegerExporter =
+        JaegerGrpcSpanExporter.newBuilder()
+            .setServiceName("example")
+            .setChannel(jaegerChannel)
+            .setDeadline(10)
+            .build();
+
+    // Get the tracer
+    TracerSdkFactory tracerFactory = OpenTelemetrySdk.getTracerFactory();
+    // Set to process the spans by the Jaeger Exporter
+    tracerFactory.addSpanProcessor(SimpleSpansProcessor.newBuilder(jaegerExporter).build());
+    // Give the name to the traces
+    this.tracer = tracerFactory.get("io.opentelemetry.example.JaegerExample");
+  }
+
+  private void myWonderfulUseCase() {
+    initTracer();
+    Span span = tracer.spanBuilder("Start my wonderful use case").startSpan();
+    span.addEvent("Event 0");
+    // execute my use case - here we simulate a wait
+    try {
+      Thread.sleep(1000);
+    } catch (InterruptedException e) {
+    }
```

Comment:
I would move both sleeps to a helper method so that the try/catch stuff doesn't blow up the sample. Something like `doOtherStuff()` would make the intent clearer.